content/src/controller/Controller.ts

```diff
@@ -407,7 +411,8 @@ export class Controller {
     // Path: /contents/:hashId/active-entities
     const hashId = req.params.hashId
 
-    const result = await this.components.deployer.getActiveDeploymentsByContentHash(hashId)
+    const result = await getActiveDeploymentsByContentHash(this.components, hashId)
+    result.filter((entityId) => !this.components.denylist.isDenyListed(entityId))
```

Comment:
I believe this filter is not mutating the result `variable`, shouldn't it be stored in another variable?

Reply:
Nice catch!

Wouldn't do the reload until this is a widely used feature